File: Java/Example/src/Calculator.java
public class Calculator {
    private double Ans;
    public Calculator(){
        this.Ans=0;
    }
    public Calculator Add(double input){
        this.Ans += input; return this;
    }
    public Calculator Sub(double input) {
        this.Ans -= input; return this;
    }
    public Calculator Multiply(double input) {
        this.Ans *= input; return this;
    }
    public Calculator Div(double input) {
        this.Ans/=input; return this;
    }
    public Calculator Exponent(double input) {
        this.Ans = Math.pow(this.Ans,input); return this;
    }
    public double Read(){
        return this.Ans;
    }
    public void Print(){System.out.println(String.format("Ans currently at: %f", this.Ans));}
}

File: Python/Calculator.py
import sys

class Calculator:
    Ans=None
    
    def __init__(self):
        self.Ans=0
        
    def Add(self,input):
        self.Ans+=input
        return self
    def Sub(self,input):
        self.Ans-=input
        return self
    def Multiply(self,input):
        self.Ans*=input
        return self
    def Div(self,input): #ternary = (min = a if a < b else b)
        self.Ans/=input
        return self
    def Exponent(self,input):
        self.Ans**=input
        return self
    Read=lambda self:self.Ans
    Print=lambda self:print(f"Ans currently at: {self.Ans}")